contrib/aws/aws-sdk-go-v2/aws/eventbridge/eventbridge.go

```diff
@@ -0,0 +1,117 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package eventbridge
```

Comment:
We should move these packages to `internal/eventbridge` so they are not publicly exposed (unless we want customers to use these directly, which I don't think its the case since customers will be using `contrib/aws/aws-sdk-go-v2/aws`, right?).

Reply:
Got it, thanks for the review!